create_binary.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+set -x
+
+pushd build_calicoctl
+docker build -t calico-build .
+popd
+
+mkdir -p `pwd`/dist
+chmod 777 `pwd`/dist
+
+docker run -v `pwd`/:/code --name pyinstaller calico-build \
```

Comment:
Sounds reasonable. I'm still trying to calibrate to how Maine Coon is making the speed versus quality trade-off.